src/js/components/Select/Select.js

```diff
@@ -100,6 +100,20 @@ const Select = forwardRef(
     const inputRef = useRef();
     const formContext = useContext(FormContext);
     const { format } = useContext(MessageContext);
+
+    const [usingKeyboard, setUsingKeyboard] = useState();
```

Comment:
wondering if we can add a comment to just say something simple of us having to do this to have better alignment with the HTML select accessibility

Reply:
Added comment for this in latest commit